starter-code/index.html

```diff
@@ -1,16 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <!-- TODO: include Bulma's style and your own style -->
-  <title>Vue.js | Bulma</title>
+  <script src="https://unpkg.com/vue@2.5.2/dist/vue.js" type="text/javascript"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/css/bulma.min.css">
```

Comment:
You including bulma with a CDN. Why did you end install it with npm then? By the way, you should never commit the `node_modules` directory.